### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by t83714**
### What this PR does

Implement the new homepage

### Checklist
- [x] There are unit tests to verify my changes are correct | Unit tests aren't applicable (delete one)
- [x] I've updated CHANGES.md with what I changed.
- [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column

### CODE REVIEW

File: magda-web-client/src/homepage-config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "twenty-1080w.jpg",
+        "twenty-2160w.jpg",
+        "twenty-2880w.jpg"
+    ],
+    tagLineTextDesktop: "An easy way to find, access and reuse public data",
+    tagLineTextMobile: "Find, share, and reuse open government data",
+    Lozenge: {
+        icon: "camara.svg",
+        text: "Explore QLD road data",
+        url: "https://search.data.gov.au/search?q=QLD%20road%20data"
```
Comment (AlexGilleran) [inline magda-web-client/src/homepage-config.js]: "Queensland road data" will work way better than "QLD road data"

Comment (t83714) [reply magda-web-client/src/homepage-config.js]: I think the text is the tiny button on the homepage. 
image
I guess use abbreviation is mainly for control its size --- correct me if I am wrong @tkeuneman 
This config file is included in repo only for reminding us its structure in future.
The actual copy is located at: https://s3-ap-southeast-2.amazonaws.com/data-gov-au-frontpage/development/homepage-config.js
(staging & production site will load file in `staging` & `production` folder).
Most changes to homepage later can be done via modifying this config on s3.

Comment (AlexGilleran) [reply magda-web-client/src/homepage-config.js]: We can leave the text as "QLD", just change the text in the URL to "queensland" :).

I don't mind if it's in S3 or here.

Comment (t83714) [reply magda-web-client/src/homepage-config.js]: Oh I see. I thought you want to change the button text.
Will change the URL now.

**Pull request merged**